### ISSUE TITLE
unity markup работает в никах

### ISSUE THREAD

**Issue by justqstn** [Sokol-Games-GmbH/PixelCombats.PublicApi#173]
очень мешает, особенно <quad>
можно сделать так, чтобы не было видно ника, нельзя было добавить в друзья и замутить игрока (вроде бы)

**Comment by forwork10** [comment]
добавить в др можно, ник не действует на эту кнопку, замутить тоже очень легко, просто в коммандах ввести блок чата "quad size"

**Comment by kkohno** [comment]
так, а где именно работает? мы оставили специально это дело в чате - там вроде безобидно относительно все это, просто там цвет часто выбирают себе игроки и спецсимволы... там вырезать честь тэгов не получится насколько знаю... если знаете как убрать частично тэги то отпишите
